cookietemple/create/github_support.py

```diff
@@ -340,11 +341,12 @@ def handle_failed_github_repo_creation(e: Union[ConnectionError, GithubException
     """
     # output the error dict thrown by PyGitHub due to an error related to GitHub
     if isinstance(e, GithubException):
-        print('[bold red]\nError while trying to create a Github repo due to an error related to Github API. See below output for detailed information!\n')
+        console.print(
```

Comment:
Again, I don't like the formatting

Reply:
I will fix this, I do not like this either ;)